js/components/charts/widget.vue

```diff
@@ -103,20 +91,13 @@ var AREA_OPTIONS = {
     ];
 
 
-module.exports = {
+export default {
     name: 'chartjs-chart',
     data: function() {
         return {
             chart: null,
-            charttype: 'Area',
             canvasHeight: null,
-            height: '300px',
-            types: {
-                Area: 'area-chart',
-                Bar: 'bar-chart',
-                StackedBar: 'bar-chart',
-                Line: 'line-chart'
-            }
```

Comment:
`types` are only useful to display the box to swap the chart types, I restored the different methods to build graphs though.